src/core/service/notifications/jobs/handler.go

```diff
@@ -87,7 +89,7 @@ func (h *Handler) Prepare() {
 // HandleNotificationJob handles the hook of notification job
 func (h *Handler) HandleNotificationJob() {
```

Comment:
Let's move this handler into the new program model as well: Put it under the directory https://github.com/goharbor/harbor/tree/master/src/server/handler

Reply:
I'll raise another PR to handle it.